plugin.go

```diff
@@ -50,6 +97,7 @@ func setupPluginTFState(p ConfigPlugin, c *Config) error {
 	if err != nil {
 		return err
 	}
+	funcs = p.ModifyFuncMap(funcs)
 	c.templateFuncs = append(c.templateFuncs, funcs)
```

Comment:
Conflicts of template functions must raise errors. So we can check conflicts at this line as below.

Then we will not need to add `ConfigPlugins.check()`.

```go
for name, _ := range funcs {
    if _, exists := c.templateFuncs[name]; exists {
        return fmt.Errorf("template function %s already exists. set func_prefix to %s plugin", name, p.Name)
    }
}
```

Reply:
https://github.com/kayac/ecspresso/pull/398/commits/435a1c9f603368328d04db74557123cb42bfa762